cypress/integration/TrialDescriptionPage/GeneralPageComponents.feature

```diff
@@ -0,0 +1,111 @@
+Feature:  As a user, I want to be able to view trial description page with all its components
```

Comment:
1. We need to test to make sure that when we go directly to a trial description without being on search. This is used by the Listing App and clinical trials pretty URLS. This adds the following steps:
* `"< Back to search results" button as link is not displayed`
* `"This clinical trial matches: \"all trials\"" does not appear below the title`
* `search criteria table is not displayed` -- ok, this one already exists...

Reply:
2. Given the nature of this page, we should be validating the metadata on the page. We are not checking:
* Title
* og:title
* og:description
* description
* Canonical URL
* og:image

Reply:
3. The viewing page supports both the NCI ID and the NCT ID as valid IDs. See:
* https://www.cancer.gov/about-cancer/treatment/clinical-trials/search/v?id=NCI-2014-01507
* https://www.cancer.gov/about-cancer/treatment/clinical-trials/search/v?id=NCT02201992

(Both of these are the same trial)

Reply:
taken care of